sitemap_utils.py

```
@@ -8,7 +8,7 @@
 import xml.etree.ElementTree as ET
 from dataclasses import dataclass
 from pathlib import Path
-from typing import List, Optional, Set, Tuple
+from typing import Dict, List, Optional, Set, Tuple
```

Ruff (0.8.2)

`typing.Dict` is deprecated, use `dict` instead

(UP035)